frontend/index.d.ts

```diff
@@ -74,6 +74,7 @@ interface CAPIType {
     webPublicationDate: Date,
     webPublicationDateDisplay: string,
     editionLongForm: string,
+    editionId: string,
```

Comment:
This could have type `'UK' | 'US' | 'INT' | 'AU'` instead of string.

Reply:
Done